acceptance-tests/src/test/resources/features/site/apiCatalogue.feature

```diff
@@ -0,0 +1,26 @@
+Feature: API Catalogue in Developer hub
+    @WIP
+    Scenario: Catalogue is rendered in Site
+        Given I navigate to "Static API Catalogue" page
+        Then I should see page titled "Test API catalogue"
+        And I can see labelled element "website.linkslist.summary" with content "These are the APIs which are currently available, or in development, from NHS Digital."
+        And I should see the list with title "Alphabetical navigation" containing:
+            | A | B | C | D | E | F | G | H | I | J | K | L | M | N | O | P | Q | R | S | T | U | V | W | X | Y | Z |
+        # Assert that index is rendered correctly
+        And I can see elements with attributes:
```

Comment:
I wonder if we could add an overloaded method here with name and text in `@Then` annotation reflecting the text in the comment. It would sit alongside and invoke the `I can see elements with attributes` method.

We could then get rid of the comment line above and have the method express the intent itself.